tests.synthetics.json

```diff
@@ -0,0 +1,10 @@
+{
+    "tests": [
+        {
+            "id": "sfs-zqk-cfq"
```

Comment:
what is this for? Are the id's specific for anything?

Reply:
these come from the DD synthetic test. i would include a comment w/ a URL but this is JSON instead of YAML.